CommonCPPCTest.cmake

```diff
@@ -121,9 +121,14 @@ foreach(FILE ${TEST_FILES})
   endif()
 endforeach()
 
+set(__CONSOLE)
+if(CMAKE_VERSION VERSION_GREATER 3.2.2)
```

Comment:
Sure? I was it as new feature in 3.2.3, but maybe they have the same relnotes for all 3.2.x